components/screens/DailyRidesScreen/DailyRidesStyles.js

```diff
@@ -29,4 +33,8 @@ export const Styles = StyleSheet.create({
         borderBottomColor: '#737373',
         borderBottomWidth: StyleSheet.hairlineWidth,
     },
+<<<<<<< HEAD
 });
+=======
+});
+>>>>>>> main
```

Comment:
It looks like we got some leftover merge conflict code here. This will need to be removed for the app to run properly.

It seems you can just `accept incoming changes` to fix this.

Reply:
good catch, I have it fixed on my machine, but did not commit the change.